central/sensor/service/connection/connection_test.go

```diff
@@ -265,14 +354,16 @@ func (s *testSuite) TestSendsAuditLogSyncMessageIfEnabledOnRun() {
 	mgrMock := clusterMgrMock.NewMockClusterManager(ctrl)
 
 	sensorMockConn := &sensorConnection{
-		clusterID:  clusterID,
-		clusterMgr: mgrMock,
+		clusterID:     clusterID,
+		clusterMgr:    mgrMock,
+		scanSettingDS: s.scanConfigDS,
```

Comment:
This shouldn't be needed in this test.

Reply:
I think it is necessary to have the DS in every test now, otherwise they panic now that central sends a scan config to the datastore